vllm/entrypoints/openai/serving_chat.py

```diff
@@ -61,8 +61,8 @@ async def create_chat_completion(
 
         request_id = f"cmpl-{random_uuid()}"
         try:
-            token_ids = self._validate_prompt_and_tokenize(request,
-                                                           prompt=prompt)
+            prompt_ids, prompt_text = self._validate_prompt_and_tokenize(
+                request, prompt=prompt)
```

Comment:
I looked through the diff but I can't find why is prompt_text needed

Reply:
On line 79, `prompt` was what was given by user (either text or token), but `generate` expected it to be text for handling echo

Reply:
Ah can you add a comment